proposals/gc/MVP.md

```diff
@@ -598,35 +598,32 @@ In particular, `ref.null` is typed as before, despite the introduction of `none`
 Casts work for both abstract and concrete types. In the latter case, they test if the operand's RTT is a sub-RTT of the target type.
 
 * `ref.test null? <heaptype>` checks whether a reference has a given heap type
-  - `ref.test null? ht : [(ref null ht')] -> [i32]`
-    - iff `ht <: tht` and `ht' <: tht` where `tht` is a common super type
-  - if `null?` is present, returns 1 for null, otherwise 0
+  - `ref.test rt : [(ref rt')] -> [i32]`
+    - iff `rt <: trt` and `rt' <: trt` for some `trt`
+  - if `rt` contains `null`, returns 1 for null, otherwise 0
```

Comment:
What does "`rt` contains `null`" mean? Is `rt` a reference type? Then it would be different in kind from `rt'`.

Reply:
Good point, I was using the wrong factorisation. Fixed.